docs/contributing/README.md

```diff
@@ -118,6 +118,71 @@ Then, run the continuous batching tests:
 python -m pytest -v -x tests/e2e -m cb
 ```
 
+## Debugging
+
+!!! tip
+    You can actually `oc edit` a pod and change the image without having the pod schedule to a different node! (useful for testing if software or hardware is the issue).
```

Comment:
```suggestion
    You can `oc edit` a pod and change the image without having the pod schedule to a different node. This can be useful for testing whether software or hardware is the issue.
```